web-ng/src/app/services/router/router.service.ts

```diff
@@ -144,8 +144,25 @@ export class NavigationService {
   /**
    * Navigate to the current URL, updating the feature id in the URL fragment.
    */
-  setFeatureId(id: string | null) {
+  setFeatureId(id: string | null): boolean {
+    if (
+      this.getFragmentParams().get(
+        NavigationService.OBSERVATION_ID_FRAGMENT_PARAM
+      ) !== null
+    ) {
+      return false;
+    }
```

Comment:
Actually, I think the right behavior here would be to remove the observationId from the URL if present, not to ignore the request.

Reply:
Removed the observationId from the URL if present.